actors/market/src/lib.rs

```diff
@@ -444,7 +460,7 @@ impl Actor {
             Ok(())
         })?;
 
-        // notify clients ignoring any errors
+        // notify clients, any failures cause the entire publish_storage_deals method to fail
```

Comment:
Please summarise here on why we want failures to abort the whole message, why it's not safe to ignore failure.